### ISSUE TITLE
[enhancement] <BLANKLINE> is not handled correctly in docs

### ISSUE THREAD

**Issue by crcrpar** [chainer/chainer#7154]
`Example`s in documentation (both stable and latest) display a ton of `<BLANKLINE>`

Search results of <BLANKLINE>:
- https://docs.chainer.org/en/stable/search.html?q=%3CBLANKLINE%3E&check_keywords=yes&area=default
- https://docs.chainer.org/en/latest/search.html?q=%3CBLANKLINE%3E&check_keywords=yes&area=default

One example is [`depth2space`](https://docs.chainer.org/en/latest/reference/generated/chainer.functions.depth2space.html?highlight=blankline).
```
>>> X = np.arange(24).reshape(1, 4, 2, 3).astype(np.float32)
>>> X.shape
(1, 4, 2, 3)
>>> X
array([[[[ 0.,  1.,  2.],
         [ 3.,  4.,  5.]],
<BLANKLINE>
        [[ 6.,  7.,  8.],
         [ 9., 10., 11.]],
<BLANKLINE>
        [[12., 13., 14.],
         [15., 16., 17.]],
<BLANKLINE>
        [[18., 19., 20.],
         [21., 22., 23.]]]], dtype=float32)
>>> y = F.depth2space(X, 2)
>>> y.shape
(1, 1, 4, 6)
>>> y.array
array([[[[ 0.,  6.,  1.,  7.,  2.,  8.],
         [12., 18., 13., 19., 14., 20.],
         [ 3.,  9.,  4., 10.,  5., 11.],
         [15., 21., 16., 22., 17., 23.]]]], dtype=float32)
```

**Comment by kmaehashi** [comment]
It seems spec of Sphinx (?) has been changed at some point. It was OK in v4 docs.
https://docs.chainer.org/en/v4.4.0/reference/generated/chainer.functions.get_item.html?highlight=blankline

**Comment by crcrpar** [comment]
1.8.0 -> 1.8.2 four months ago.

**Comment by beam2d** [comment]
@Crissman What is the status of this issue?

**Comment by toslunar** [comment]
It seems fixed from the docs for Chainer v7.0.0b4.
